docs/Course Wiki/ECE Course Offerings/ECE374B.md

```diff
@@ -0,0 +1,69 @@
+
+
+## Instructors
+In recent years (as of Fall 2023), Professor Nickvash Kani has taught the B section of the course. 
+
+## Course Tips
+Most people will find that working in a group is very helpful for learning the content and getting through the course. It is important to find a group you work well with, and don't be afraid to leave a group that does not work well together. Signing up for the class with people you know you work well with already is a good way to ensure a successful group.
+
+Once in a group, the recommended way to approach group homeworks is to do them separately, then meet up and discuss and compare your solutions. Homeworks don't always need to be typed up, but if you have messy handwriting it might save you some points lost for lack of clarity. Homework problems sometimes show up on exams, so doing hws separately guarantees that you know all of the problems on each homework.
+
+Go to discussions and make sure to write down the problems so you know them when they come up in the homeworks.
+
+Make sure to show your work on exams, as there are decent amounts of partial credit offered.
+
+
+## Life After
+ECE374 opens students up to most of the upper level courses in the [CS department](../../CS%20Course%20Offerings/). If you enjoyed the 2nd section of the course, the most obvious follow up would be [CS473 - Algorithms](../CS%20Course%20Offerings/CS473.md). If you liked the first and third sections, you should take [CS475 - Formal Models of Computation](../CS%20Course%20Offerings/CS475.md).
+
+## Infamous Topics
+- Language Transformations: The most difficult topic from the first section. You must know it well because there will be a problem about it on MT1. TAs usually can explain language transformations well. There are also good examples in the textbook.
+- Dynamic Programming: Good to know for getting a Software Engineering Internship.
+- NP hardness: Proofs on NP-Hardness can get confusing quick. It is helpful to draw out diagrams in order to not get confused on which direction the proof goes.
+
+## Resources
++ The official course website: [https://ecealgo.com/](https://ecealgo.com/)
++ Prof Jeff Erickson's textbook and notes: [https://jeffe.cs.illinois.edu/teaching/algorithms/](https://jeffe.cs.illinois.edu/teaching/algorithms/)
++ 374A section website: [https://courses.engr.illinois.edu/cs374al1/sp2023/](https://courses.engr.illinois.edu/cs374al1/sp2023/)
```

Comment:
I would change this to the FA23 version if you're going to leave it on here (although right now we do not know what that link is...)
Just delete it to save some hassle. You can keep the B-section link though.